bin/clang/pragmas.cc

```diff
@@ -330,6 +332,7 @@ SSTPragma::getMap(SourceLocation loc, const std::list<clang::Token>& tokens)
         sstr.str("");
         allArgs[argName] = std::move(argList);
         argList = {};
+        hitParen = false;
```

Comment:
This Paren logic is confusing, perhaps it could be clean up a bit so that it is easier to follow.  For example how can parenDepth != 0, but hitParen is false?

Reply:
I have added comments that should clear up what's going on